Tests/Tests/HttpRequestableSpec.swift

```diff
@@ -26,7 +26,10 @@ class HttpRequestableSpec: QuickSpec {
             }
 
             afterEach {
-                httpRequestable = nil // force deallocation of .httpSessionMock
+                // force deallocation of httpSessionMock
```

Comment:
We are not forcing deallocation here. Just simply clearing the state of `httpRequestable`. I think you can just remove this comment